mobile/library/kotlin/io/envoyproxy/envoymobile/mocks/MockEnvoyEngine.kt

```diff
@@ -13,9 +13,10 @@ import io.envoyproxy.envoymobile.engine.types.EnvoyStringAccessor
 internal class MockEnvoyEngine : EnvoyEngine {
   override fun runWithConfig(envoyConfiguration: EnvoyConfiguration?, logLevel: String?): Int = 0
 
-  override fun runWithTemplate(
+  override fun performRegistration(envoyConfiguration: EnvoyConfiguration) = Unit
```

Comment:
I don't speak Kotlin, but I notice that this method has `= Unit` and the other two have `= 0`. Do they mean the same thing?

Reply:
It's because `runWithConfig` returns an integer and `runWithConfig` returns "nothing" (aka `void` / `Void` / `Unit`).